lib/matplotlib/dviread.py

```diff
@@ -756,29 +747,23 @@ def __init__(self, filename):
         _log.debug('opening tfm file %s', filename)
         with open(filename, 'rb') as file:
             header1 = file.read(24)
-            lh, bc, ec, nw, nh, nd = \
-                struct.unpack('!6H', header1[2:14])
+            lh, bc, ec, nw, nh, nd = struct.unpack('!6H', header1[2:14])
             _log.debug('lh=%d, bc=%d, ec=%d, nw=%d, nh=%d, nd=%d',
                        lh, bc, ec, nw, nh, nd)
             header2 = file.read(4*lh)
-            self.checksum, self.design_size = \
-                struct.unpack('!2I', header2[:8])
+            self.checksum, self.design_size = struct.unpack('!2I', header2[:8])
             # there is also encoding information etc.
             char_info = file.read(4*(ec-bc+1))
-            widths = file.read(4*nw)
-            heights = file.read(4*nh)
-            depths = file.read(4*nd)
-
+            widths = [*struct.iter_unpack('!i', file.read(4*nw))]
```

Comment:
I have no idea of dvi internals. How many are these? Just ~10 or thousands? In case of the latter, this might have a performance impact.

Isn't this just `struct.unpack(f'!{nw}i', file.read(4*nw))` which seems simple enough?

Reply:
Indeed, that's an even simpler approach :+1:

Reply:
Such cases let you really appreciate f-strings. How could we have lived without them? 😄